db/impex/master/pom.xml

```diff
@@ -40,7 +40,6 @@
     <profile>
       <id>master</id>
       <properties>
-        <impex.username>RICE_${project.version.sanitized}</impex.username>
```

Comment:
I don't see any reason why we would need to remove this property... I'd leave it in.